avocado-setup.py

```diff
@@ -730,9 +730,12 @@ def parse_test_config(test_config_file, avocado_bin, enable_kvm):
         test_name_list = []
         for test_suite in Testsuites_list:
             test_name_list.append(Testsuites[test_suite].name)
-        test_name_list.append(Testsuites[Testsuites_list[0]].runlink.split('\n')[0])
-        test_name_list.append(Testsuites[Testsuites_list[0]].runlink.split('\n')[1])
-        longest_name_length = len((sorted(test_name_list, key=len)[-1])) + 5
+            if Testsuites[test_suite].runlink:
```

Comment:
previously  `test_name_list` got appended for only one runlink, irrespective of how many ever testsuites, now it does extend for every testsuite?, any reason for this change?

Reply:
@sathnaga Reason why we did this:

We are finding the length needed for column 1, which is the longest of:
1. TestSuiteName
2. job.log path
3. result link

Now, 2 and 3 are static for one wrapper run, so we earlier added only the 1st testsuite's data there. Now, we are not sure which testsuite will have link and which ones will not, so handled in a generic way by having all testsuite's data added, which adds more than needed data in the test_suite list, but solves our issues.

Reply:
Good, capture this explanation in commit message, which will help us in future...